packages/core/namespace.ts

```diff
@@ -55,8 +56,13 @@ type MetaDataProperty =
 
 type CubeCreatorNamespace = Record<CubeCreatorClass | CubeCreatorProperty | OtherTerms | MetaDataProperty, NamedNode>
 
+prefixes.scale = 'http://ns.bergnet.org/cube/scale/'
+prefixes.freq = 'http://purl.org/cld/freq/'
+prefixes.cube = 'http://ns.bergnet.org/cube/'
```

Comment:
I completely forgot. It is possible to add a prefix to `@zazuko/rdf-vocabularies`. `rdf-string` will also honor them when interpolating named nodes